### PR TITLE
Added reading chelpg charges to ORCA

### PR DESCRIPTION
**Pull request by richardjgowers**
This adds reading chelpg charges from ORCA.  I've got a test file which has these in, but I couldn't figure out where to add a test just for the ORCA parser?  I'll put up a PR for the docs on the doc repo

### CODE REVIEW

**Comment by berquist** [comment]
Thanks for the patch!

>  I've got a test file which has these in, but I couldn't figure out where to add a test just for the ORCA parser?

If it's the usual test molecule that we use (divinylbenzene), then what you can do is add a test in `test/data/testSP.py`, probably as a method under `GenericSPTest`, then add `@skipForParser` decorators for all parsers except ORCA. That way we can use the test in the future for Gaussian and Q-Chem, both of which also support CHELPG charges.

If it's a one-off calculation, then we have a separate repository for regression and contributed tests: https://github.com/cclib/cclib-data.

Also, would it be possible to turn parsing all 3 charge methods (Mulliken, Lowdin, CHELPG) into a single class function? The code is basically identical between the 3. See the Q-Chem parser below:

https://github.com/cclib/cclib/blob/e7cc329adcdcf1347c686b826cab37bf07dd0a0e/src/cclib/parser/qchemparser.py#L162

**Comment by richardjgowers** [comment]
@berquist thanks, I refactored the charge reading, I'll look at adding the test to the other repo

File: src/cclib/parser/orcaparser.py [hunk 1]
```diff
@@ -842,6 +809,48 @@ def extract(self, inputfile, line):
                 polarizability.append(line.split())
             self.polarizabilities.append(numpy.array(polarizability))
 
+    def parse_charge_section(self, line, inputfile, chargestype):
```
Comment (berquist) [inline src/cclib/parser/orcaparser.py]: Can you add a docstring?

**Comment by berquist** [comment]
Great, thank you!

**Pull request merged**